src/krate/mod.rs

```diff
@@ -280,20 +279,16 @@ impl<'a> NewCrate<'a> {
 }
 
 impl Crate {
```

Comment:
Could you add doc comments to the `by_name`, `with_name`, and `all` functions in this struct?